### PR TITLE
nmtui: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages.pl/linux/google-chrome-stable.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# google-chrome-stable
+
+> To polecenie jest aliasem dla `chromium`.
+> Więcej informacji: <https://chrome.google.com>.
+
+- Zobacz dokumentację oryginalengo polecenia:
```
Comment (spageektti) [inline pages.pl/linux/google-chrome-stable.md:6]:
```suggestion
- Zobacz dokumentację oryginalnego polecenia:
```

**Comment by spageektti** [comment]
I resolved merge conflicts (#14345 is merged)

> This PR includes files from #14345

File: pages.pl/linux/nmtui-connect.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# nmtui-connect
+
+> To polecenie jest aliasem alias `nmtui connect`.
```
Comment (spageektti) [inline pages.pl/linux/nmtui-connect.md:3]:
```suggestion
> To polecenie jest aliasem `nmtui connect`.
```

File: pages.pl/linux/nmtui-edit.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# nmtui-edit
+
+> To polecenie jest aliasem alias `nmtui edit`.
```
Comment (spageektti) [inline pages.pl/linux/nmtui-edit.md:3]:
```suggestion
> To polecenie jest aliasem `nmtui edit`.
```

File: pages.pl/linux/nmtui-hostname.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# nmtui-hostname
+
+> To polecenie jest aliasem alias `nmtui hostname`.
```
Comment (spageektti) [inline pages.pl/linux/nmtui-hostname.md:3]:
```suggestion
> To polecenie jest aliasem `nmtui hostname`.
```

File: pages.pl/linux/cacafire.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# cacafire
+
+> Wyświetl animowany ogień ASCII.
+> Więcej informacji: <https://packages.debian.org/sid/caca-utils>.
+
+- Wyświetl animowany ogień ASCII:
```
Comment (spageektti) [inline pages.pl/linux/cacafire.md:6]:
```suggestion
- Wyświetl ogień ASCII:
```
to match the original page

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**